google-cloud-tasks/README.md

```diff
@@ -28,41 +29,4 @@ $ gem install google-cloud-tasks
   to see the full list of Cloud APIs that we cover.
 
 [Client Library Documentation]: https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-tasks/latest/google/cloud/tasks/v2beta2
-[Product Documentation]: https://cloud.google.com/tasks
-
-## Enabling Logging
```

Comment:
Why is this section removed?

Reply:
Was overly conservative during the generation. Re-added newest README.